README.md

```diff
@@ -1,3 +1,30 @@
+# About this Fork
+
+This repository is a fork of the official TypeORM repository. It is customized and maintained for use in the [121 platform](https://github.com/global-121/121-platform).
+
+## CI & CD
+
+Originally, tests ran in CircleCI. In this fork, they have been migrated to run in GitHub Actions.
+
+### How to Get Your Change to 121
+
+**TypeORM Repo:**
+
+- **Code Changes:**
+  - Follow the same rebase process as on [121](https://github.com/global-121/121-platform/wiki/Git-rebase).
+  - Be aware that when you create a PR, you should point it towards the `master` branch of the fork and not the `master` branch of the original repository.
+
+- **Release:**
+  - Follow the same process as described [here](https://github.com/global-121/121-platform?tab=readme-ov-file#releases) for the 121 platform. Just use SemVer instead of CalVer. To generate a new version, you can use:
```

Comment:
This doesn't make sense to me. Is it really the same process? I think there are extra steps there that don't apply here. Also, if that documentation diverges from what we need here, we will never know.

I would rather be explicit here and write down what needs to be done for this package, without referring to the 121-platform.

Reply:
Good points :)